api/app/signals/apps/api/validators/source.py

```diff
@@ -29,19 +28,15 @@ def __call__(self, value, serializer_field):  # noqa: C901
         if not Source.objects.filter(name__iexact=value, is_active=True).exists():
             raise ValidationError('Invalid source given, value not known')
 
-        # No need to check the given source this will be overwritten when creating the child Signal
```

Comment:
Keep (an updated version) of this comment line, something like:

"""For child Signal instances the `source` attribute will be set to `settings.API_TRANSFORM_SOURCE_OF_CHILD_SIGNAL_TO` so we do not have to check it here."""